2.0/s2i/bin/run

```diff
@@ -7,9 +7,21 @@ function is_puma_installed() {
 
 set -e
 
+function check_number() {
+  if [[ ! "$2" =~ ^[0-9]+$ ]]; then
+    echo "$1 needs to be a non-negative number"
+    exit 1
+  fi
+}
+check_number PUMA_WORKERS     "${PUMA_WORKERS:-0}"
+check_number PUMA_MIN_THREADS "${PUMA_MIN_THREADS:-0}"
+check_number PUMA_MAX_THREADS "${PUMA_MAX_THREADS:-0}"
```

Comment:
We discussed this off-line. My main reason for leaving it here is that I think of the env vars as an interface to our image and we do validate vars in bash for example in the MySQL image (and I assume other as well).